packages/synapse-react-client/src/components/QueryWrapper/QueryWrapperLoadingScreen.tsx

```diff
@@ -1,20 +1,16 @@
-import { useQueryContext } from '../QueryContext'
-import { useAtomValue } from 'jotai'
-import { isLoadingNewBundleAtom, tableQueryDataAtom } from './QueryWrapper'
 import { Box, LinearProgress, Typography } from '@mui/material'
 import React from 'react'
 
-export default function QueryWrapperLoadingScreen() {
-  const { asyncJobStatus } = useQueryContext()
-  const isLoadingNewBundle = useAtomValue(isLoadingNewBundleAtom)
-  const data = useAtomValue(tableQueryDataAtom)
-  if (data || !isLoadingNewBundle) {
-    return <></>
-  }
+type QueryWrapperLoadingScreenProps = { progressMessage?: string }
+
+export default function QueryWrapperLoadingScreen(
```

Comment:
Made this component 'dumb' so it just shows the progressMessage passed as props.

Keep in mind that since we're firing ~2 queries at any time (one for rows, one for metadata/stats), there were actually two progress messages that could differ over time.

The component(s) that actually use this loader will pass the progress message that corresponds to the data that the component is actually waiting for, so it won't do weird things like
* Say "Complete" when one of the queries is not actually complete
* Say "Complete" and revert to some 'incomplete' message as a second query updates after the first query completes.